package.json

```diff
@@ -74,6 +74,7 @@
     "@types/react": "^18.2.22",
     "@types/react-dom": "^18.2.7",
     "browserslist": "^4.16.15",
+    "cmake-js/axios": "^1.6.0",
```

Comment:
do we know what this is used for?

Reply:
Not sure how to interpret this but `yarn why cmake-js` shows 
``` 
=> Found "cmake-js@7.2.1"
info Has been hoisted to "cmake-js"
info Reasons this module exists
   - "workspace-aggregator-43cb8e75-af3d-4e3b-9a57-f6c44f620bea" depends on it
   - Specified in "dependencies"
   - Hoisted from "_project_#cmake-js"
  ```